CI-CD DOCUMENTATION.md

```diff
@@ -1,3 +1,6 @@
+# Built with love by Pipeline Foundation.
```

Comment:
Please leave the credits only at the bottom of the md file.

Reply:
Done